packages/code-du-travail-frontend/src/outils/indemniteLicenciement/ccn/0016/CategoriePeriod.js

```diff
@@ -1,5 +1,9 @@
 import React from "react";
-import { SectionTitle, Input, InlineError } from "../../stepStyles";
+import {
+  SectionTitle,
+  Input,
+  InlineError
+} from "../../components/styledComponents";
```

Comment:
I would prefer to call it uiComponents that styledComponents which is tied to a library name